README.md

```diff
@@ -1,48 +1,53 @@
-# robosub-ros-docker
+# Duke Robotics Club - RoboSub ROS Docker
+
 ![Build Status](https://github.com/dukerobotics/robosub-ros-docker/workflows/build/badge.svg)
 ![Docker Pulls](https://img.shields.io/docker/pulls/dukerobotics/robosub-ros)
 
-Docker image for ROS system to control a robot for the RoboSub Competition
+We are a Duke University club competing in the RoboSub Competition. Check out [our website](https://duke-robotics.com).
+
+This repo contains our Dockerfiles and instructions for how to build them, in support of our main [robosub-ros](https://github.com/DukeRobotics/robosub-ros) repo. If you're looking to use these images to run our code, check out the main repo.
+
+Our images are automatically built and pushed to [our Docker Hub repo](https://hub.docker.com/r/dukerobotics/robosub-ros) using GitHub CI/CD.
+
 
-## Summary
-Docker is a technology that allows encapsulation of the running environment. A docker container is like a tool that can be used to run code, irrespective of the computer. For more information, go [here](https://www.docker.com/resources/what-container).
+## Our images
 
-As an example scenario, imagine that you made a python script and you want your friend to be able to run it. This python script relies on a couple different packages, that you downloaded on to your computer through various means. Now you send it to your friend and they try and run it, but then tell you it won't run, because they don't have any of the packages necessary. You can solve this in two ways. The first is to tell them how to install everything, send them links to where you got the packages, etc. Oh also, they're running Windows and you're on Ubuntu. You then realize that one of the packages only exists for Windows, and you start searching for a replacement.
-Now the second option: Docker. What you can do is create a Dockerfile that inherits from Ubuntu, and then you can write out a series of instructions containing all the steps you followed to create your environment. For example, you can put all the `apt-get install`s in this Dockerfile. Then you can build this Dockerfile into an image, and put the image online. Now, all your friend on Windows has to do is download your image, create a container, and use the container to run your code. This means that your friend does not need to do any installation, and also runs the code in the exact same environment as you, despite being on a different computer.
+- `base` - built from NVIDIA's images to fix their incorrect architecture tagging
```

Comment:
Technically, the base image for amd64 is the stock image of Ubuntu that is published by Docker. However, I don't know how to better word this.

Reply:
I tried a new way to word, let me know if I should try again.